README.md

```diff
@@ -1,18 +1,72 @@
 # Community Services Demographic Growth Tool
 
-A Shiny app that combines Community activity recorded in 2022-23, from the NHS England Community Services Dataset (CSDS), as a baseline, with [ONS subnational population projections](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationprojections/bulletins/subnationalpopulationprojectionsforengland/previousReleases) to 2043, to provide projected usage of community services.
+A [Shiny app][tool] that combines Community activity recorded in 2022-23, from the NHS England Community Services
+Dataset ([CSDS][csds]), with [ONS subnational population projections][ons] to 2043, to show the projected usage of
+community services.
+
+For this app, projections are calculated by financial year, to match the basis period of the CSDS data.
 
-Projections are reported by financial year.
 The app provides breakdowns of the data:
 
-* by ICB, as well as providing national figures
-* by age group.
+* by ICB, as well as providing national figures for England
+* by age group
+* by service (eg Physiotherapy)
+* as a rate per individual patient
+* as a rate per 1,000 population of the ICB (based on current boundaries).
+
+The app can show activity in terms of number of contacts (appointments etc), or in terms of the number of individual
+patients using the services.
+
+## Running the app locally
+
+Clone the package repository and check you have all the required R dependencies installed (see the DESCRIPTION file for
```

Comment:
```suggestion
Clone the package repository and check you have all the required R dependencies installed (see the `DESCRIPTION` file for
```

Perhaps just suggest they install the package (and give code to do that) rather than directing them to the `DESCRIPTION`?

Reply:
Somehow I had not actually managed to realise that you can install this package and run `run_app()`, which is exported... doh! I was still in the paradigm that you need to run it from within the package folder itself.